### DISCUSSION TITLE
Digital/2024/lab2/simulation/

### DISCUSSION THREAD

**Discussion by giscus** [wintermelon008/USTC-DigitalLab-Discussion#37] [Announcements]
# Digital/2024/lab2/simulation/

https://soc.ustc.edu.cn/Digital/2024/lab2/simulation/

**Comment by Sophon3266** [comment]
在3.3步中，点击 Run Linter后Vivado就闪退了，是怎么回事呢

**Reply by wintermelon008** [reply]
可以看一下程序有没有提供闪退的信息。如果是虚拟机中的2023.1版本，可能是因为内存不足；如果是自己的电脑，可能是因为中文路径。如果都不是的话，可以问一下自己组的助教~

**Reply by Sophon3266** [reply]
没事了，忘记之前把电脑名字改成中文了:(
